src/Chunk.ts

```diff
@@ -344,7 +344,9 @@ export default class Chunk {
 		}
 
 		if (name === 'default') {
-			return;
+			// we found a missing default export, so
+			// let the module create it's own shim.
+			return { variable: module.traceExport(name), module };
```

Comment:
This is only possible under shimMissingExports right? Because otherwise there would have been an earlier throw?

Is it not possible to ensure this is populated earlier in the Module trace process? Is the issue that shimMissingExports is not populating module.reexports?